File: app/src/main/java/com/book/novel/adapter/KeyWordAdapter.java
package com.book.novel.adapter;

import com.book.ireader.ui.base.adapter.BaseListAdapter;
import com.book.ireader.ui.base.adapter.IViewHolder;
import com.book.novel.adapter.view.KeyWordHolder;

/**
 * Created by newbiechen on 17-6-2.
 */

public class KeyWordAdapter extends BaseListAdapter<String> {
    @Override
    protected IViewHolder<String> createViewHolder(int viewType) {
        return new KeyWordHolder();
    }
}

File: app/src/main/java/com/book/novel/adapter/KeyWordAdapter.kt
package com.book.novel.adapter

import com.book.ireader.ui.base.adapter.BaseListAdapter
import com.book.ireader.ui.base.adapter.IViewHolder
import com.book.novel.adapter.view.KeyWordHolder

/**
 * Created by newbiechen on 17-6-2.
 */

class KeyWordAdapter : BaseListAdapter<String>() {
    override fun createViewHolder(viewType: Int): IViewHolder<String> {
        return KeyWordHolder()
    }
}
